test/inductor/test_torchinductor.py

```diff
@@ -13400,6 +13400,35 @@ def test_special_polygamma(self):
         self.common(fn, (1, x))
         self.common(fn, (2, x))
 
+    @skip_if_halide  # only 32-bit indexing
+    @largeTensorTest("16GB", inductor=True)
+    def test_split_reduction_with_int64_size(self):
+        if torch._inductor.config.cpu_backend == "triton":
+            raise unittest.SkipTest(
+                "Fail for triton cpu backend with error: https://gist.github.com/shunting314/a873fb32b6b7b5a437f44280ae86839f"
+            )
+
+        if self.device == "cpu":
+            raise unittest.SkipTest(
+                "The test fails some times on CI: "
+                "https://github.com/pytorch/pytorch/actions/runs/15333913377/job/43153170162. "
```

Comment:
@leslie-fang-intel this test for large tensor fails sometimes in CI. I can not repro on my develop machine though. I logged into the CI machine for the failed test mentioned in the message, one difference I see is, the CI machine generate code using 8 OMP threads, while my dev server generates code using 192 threads.

Initially I thought the issue is caused by int32 overflow, but the index in the generated code is already int64_t. 

Generated code copied from CI machine with minor debugging change: https://gist.github.com/shunting314/daad71e283507f2502e41cc7e781faf4

Can you team take a further look?